README.md

```diff
@@ -32,15 +32,20 @@ First since we'll launch kind with GPU support, ensure that the following prereq
    Container Runtime to use volume mounts to select devices to inject into a
    container.
 
+1. Show the current GPU information of the machine
```

Comment:
I think this step is covered at  https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html)

Reply:
Yes, but would it be helpful to have it in the root README. For example, I was following the instructions in this README in my first attempt.